rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/InfrastructureManager.java

```diff
@@ -89,22 +89,21 @@
     /** class' logger */
     protected static final Logger logger = Logger.getLogger(InfrastructureManager.class);
 
-    /** delay that is waited at startup of the scheduler to initialize the
-     * variables and persist them
-     */
-    private static final int PERSISTED_INFRA_VARIABLES_INIT_DELAY = 2000;
-
     /** manager's node source */
     protected NodeSource nodeSource;
 
```

Comment:
@fviale please notice below the changes where:
- before:  Map<String, RMDeployingNode> were persisted
- after:  Set<String> are persisted + Map<String, RMDeployingNode> are kept in memory